CONTRIBUTING.md

```
@@ -1 +1,135 @@
+# Contributing to n8n-kit 
+
+Thank you for your interest in contributing to n8n-kit! This guide will help you get started with contributing to the project.
+
+## Getting Started
+
+### Prerequisites
+
+- **Bun >= 1.2.20** - Required for expect type features
+
+### Commands
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)